Makefile

```diff
@@ -9,7 +9,7 @@ install-base:
 	poetry install
 
 install: install-base
-	poetry install -E postgres -E redshift -E mongo -E redis
+	poetry install -E postgres -E redshift -E mongo -E redis -E mysql
 
 ## Test
 test-base:
```

Comment:
the without-extras tests will probbally require you to update the `setup.cfg` with `mysql` as a new mark, and in this `test-base` add `and not mysql`.